client/makelunch.js

```diff
@@ -32,7 +42,7 @@ Meteor.startup(function () {
       path:'/editmeal/:_id',
       data: function () {
         var meal = Meals.findOne(this.params._id)
-        console.log(meal)
+        if (!meal) return false
```

Comment:
This is a bugfix for a problem I don't quite understand - sometimes after a server restart it doesn't find a meal, which causes a crash and a blank page. If it avoids the crash (by returning false for a null meal) then it tries again and finds it, and everything works out fine.